src/Storage/Device.php

```diff
@@ -271,7 +271,7 @@ abstract public function getPartitionTotalSpace(): float;
      * Get all files and directories inside a directory.
      *
      * @param  string  $dir Directory to scan
-     * @return string[]
+     * @return array
```

Comment:
Let's use `array<string>` (or whatever the actual array type is)

Reply:
On Local device i it is string[] (string of files)
On s3 it is an array, with one of the entries named 'Contents' which is the array of files
So not sure how to make this dynamic?
On S3 we need the extra entries.
To make it consistent I would change the Local device to be an array as well?

Reply:
Done!